### PR TITLE
Bid cancel reject and sharp removal

### PR DESCRIPTION
**Pull request by qqTYXn7**

### CODE REVIEW

**Comment by HenkSwardt** [comment]
https://github.com/particl/particl-market/issues/223

File: src/core/helpers/ImageProcessing.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import * as _ from 'lodash';
 import sharp = require('sharp');
```
Comment (rynomster) [inline src/core/helpers/ImageProcessing.ts]: Doesn't get rid of sharp

File: src/core/helpers/ImageProcessing.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import * as _ from 'lodash';
...
 import { ImageVersion } from './ImageVersion';
 import {MessageException} from '../../api/exceptions/MessageException';
+import Jimp = require('jimp');
```
Comment (rynomster) [inline src/core/helpers/ImageProcessing.ts]: Use `import * as Jimp from 'jimp' `

**Comment by qqTYXn7** [comment]
Fix'd.
Anything else?

**Comment by qqTYXn7** [comment]
Don't merge yet

**Comment by qqTYXn7** [comment]
Sorry about that, I was rushing and did a half assed job.
Should be working now

**Comment by rynomster** [comment]
Sharp is still in master.. @HenkSwardt please can we get this sorted out asap? I'm leaving in less than an hour :(

**Comment by qqTYXn7** [comment]
Did you mean remove sharp from the tests? Working on it

**Comment by qqTYXn7** [comment]
Sharp has been removed, albeit I don't know if the tests pass yet

**Pull request merged**